pkg/api/pod/util.go

```diff
@@ -233,6 +233,11 @@ func visitContainerConfigmapNames(container *api.Container, visitor Visitor) boo
 	return true
 }
 
+// IsPodRunning returns true if the phase of the pod is PodRunning; false otherwise.
```

Comment:
I'd argue that this check is so simple we don't need to create a helper here, it's different when looking at conditions. Ie, I'd drop this helper.